From 39642c26fb08db515272c070c512ed477df0d228 Mon Sep 17 00:00:00 2001
From: Marcel Alexandru Nitan <nitan.marcel@protonmail.com>
Date: Thu, 16 Jun 2022 16:09:35 +0300
Subject: [PATCH] Disable cinny's notification sound

This is controlled by the matrix push server now and not by cinny
---
 src/client/state/settings.js | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/src/client/state/settings.js b/src/client/state/settings.js
index 7454be3..71521a5 100644
--- a/src/client/state/settings.js
+++ b/src/client/state/settings.js
@@ -136,12 +136,13 @@ class Settings extends EventEmitter {
   }
 
   getIsNotificationSounds() {
-    if (typeof this.isNotificationSounds === 'boolean') return this.isNotificationSounds;
+    return false;
+    // if (typeof this.isNotificationSounds === 'boolean') return this.isNotificationSounds;
 
-    const settings = getSettings();
-    if (settings === null) return true;
-    if (typeof settings.isNotificationSounds === 'undefined') return true;
-    return settings.isNotificationSounds;
+    // const settings = getSettings();
+    // if (settings === null) return true;
+    // if (typeof settings.isNotificationSounds === 'undefined') return true;
+    // return settings.isNotificationSounds;
   }
 
   setter(action) {
-- 
2.30.2

